packages/@ourworldindata/grapher/src/slopeCharts/SlopeChart.tsx

```diff
@@ -493,43 +511,163 @@ export class SlopeChart
             : 0
     }
 
-    @computed private get maxLabelWidth(): number {
-        // TODO: copied from line legend
-        const fontSize =
-            GRAPHER_FONT_SCALE_12 * (this.manager.fontSize ?? BASE_FONT_SIZE)
-        return (
-            max(
-                this.series.map(
-                    (series) =>
-                        Bounds.forText(series.seriesName, { fontSize }).width
-                )
-            ) ?? 0
+    @computed get maxLineLegendWidth(): number {
+        return 0.25 * this.innerBounds.width
+    }
+
+    @computed get lineLegendFontSize(): number {
+        return LineLegend.fontSize({ fontSize: this.fontSize })
+    }
+
+    @computed get lineLegendYRange(): [number, number] {
+        const top = this.bounds.top
+
+        const bottom =
+            this.bounds.bottom -
+            // leave space for the x-axis labels
+            BOTTOM_PADDING +
+            // but allow for a little extra space
+            this.lineLegendFontSize / 2
+
+        return [top, bottom]
+    }
+
+    @computed private get lineLegendPropsCommon(): Partial<LineLegendProps> {
+        return {
+            yAxis: this.yAxis,
+            maxWidth: this.maxLineLegendWidth,
+            fontSize: this.fontSize,
+            isStatic: this.manager.isStatic,
+            yRange: this.lineLegendYRange,
+            verticalAlign: VerticalAlign.top,
+            showTextOutlines: true,
+            textOutlineColor: this.backgroundColor,
+            focusedSeriesNames: this.focusedSeriesNames,
+            onMouseOver: this.onLineLegendMouseOver,
+            onMouseLeave: this.onLineLegendMouseLeave,
+        }
+    }
+
+    @computed private get lineLegendPropsRight(): Partial<LineLegendProps> {
+        return {
+            xAnchor: "start",
+            fontWeight: 700,
+        }
+    }
+
+    @computed private get lineLegendPropsLeft(): Partial<LineLegendProps> {
+        return {
+            xAnchor: "end",
+            fontWeight: this.showSeriesNamesInLineLegendLeft ? 700 : undefined,
```

Comment:
I don't think you need this, since it's part of the line legend code anyhow, right?

Reply:
It isn't currently but I can make it so :)